universal/store/createStore.js

```diff
@@ -1,6 +1,9 @@
 import { applyMiddleware, compose, createStore } from 'redux'
 import thunk from 'redux-thunk'
 import makeRootReducer from './reducers'
+import createLogger from 'redux-logger'
+
+const logger = createLogger()
```

Comment:
we should probably create the logger within that if statement so that this isn't executed when its not needed.